README.md

```diff
@@ -14,6 +14,18 @@ result = elevated_runner.powershell_elevated('dir', 'Administrator', 'password')
 puts "Std out: #{result.output}"
 ```
 
+### Impersonating a service account
+By passing a `nil` password, winrm-elevated will assume that the command should run as a service account:
```

Comment:
The term "service account" seems a bit misleading because really its the SYSTEM account, right?